selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/AdminOfParentOrganizationViewsTest.java

```diff
@@ -24,52 +25,44 @@
 import com.google.inject.Inject;
 import com.google.inject.name.Named;
 import java.util.ArrayList;
-import org.eclipse.che.commons.lang.NameGenerator;
 import org.eclipse.che.multiuser.organization.shared.dto.OrganizationDto;
 import org.eclipse.che.selenium.core.client.TestOrganizationServiceClient;
 import org.eclipse.che.selenium.core.user.TestUser;
 import org.eclipse.che.selenium.pageobject.dashboard.CheMultiuserAdminDashboard;
 import org.eclipse.che.selenium.pageobject.dashboard.NavigationBar;
 import org.eclipse.che.selenium.pageobject.dashboard.organization.OrganizationListPage;
 import org.eclipse.che.selenium.pageobject.dashboard.organization.OrganizationPage;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 /**
- * Test validates organization views for organization admin.
+ * Test validates organization views for admin of the organization.
  *
  * @author Ann Shumilova
  */
-public class AdminOrganizationTest {
-  private static final Logger LOG = LoggerFactory.getLogger(AdminOrganizationTest.class);
+public class AdminOfParentOrganizationViewsTest {
```

Comment:
Are we really need **views** word in test name? IMHO it can be omitted so as _view_ is just another name of window with organization page, and it's obvious that selenium tests are testing GUI which consists of windows/views.